console/templates/layout.html

```diff
@@ -31,6 +31,11 @@
                 <a href="{{ href|e }}">{{ caption|e }}</a>
               </li>
               {% endfor %}
+              {% if current_user.email=="admin" %}
```

Comment:
Wouldn't just check that the user's email is "admin" as that's probably pretty easy to spoof and elevate yourself to admin permissions. Should be checking permissions on the actual user from the db (admin user should be flagged differently)